birdhouse/components/README.rst

```diff
@@ -821,6 +821,42 @@ Usage
 The service is available at ``${BIRDHOUSE_PROXY_SCHEME}://${BIRDHOUSE_FQDN_PUBLIC}/jupyter``. Users are able to log in to Jupyterhub using the
 same user name and password as Magpie. They will then be able to launch a personal jupyterlab server.
 
+GPU support for jupyterlab containers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If the host machine has GPUs and you want to make them available to the docker containers running Jupyterlab:
+
+1. ensure that the GPU drivers on the host machine are up to date
+2. install the `NVIDIA container toolkit`_ package on the host machine
+3. configure docker to use the NVIDIA as a container run-time (See the "configure docker" section in `NVIDIA container toolkit`_)
+4. restart docker for the changes to take effect
+5. add the following to the ``JUPYTERHUB_CONFIG_OVERRIDE`` variable in your local environment file:
+
+.. code-block:: shell
+
+    export JUPYTERHUB_CONFIG_OVERRIDE='
```

Comment:
Should append to `JUPYTERHUB_CONFIG_OVERRIDE` instead because you never know if that variable is used somewhere before.